arch/common/isr_tables.c

```diff
@@ -15,11 +15,11 @@
 struct int_list_header {
 	uint32_t table_size;
 	uint32_t offset;
-#if IS_ENABLED(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
+#if defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
```

Comment:
why not use `#ifdef` for single Kconfig entries?

Reply:
Because then I would have needed to change the replacement depending on whether there were `#elif` terms or not.
This is probably why the recommendation on `IS_ENABLED` is to use `#if defined()` as opposed to `#ifdef`.